### PR TITLE
New package: Microsoft.PowerQuery version 2.62.5222.761

### PR DESCRIPTION
**Pull request by noraa-junker**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123306)

### CODE REVIEW

**Comment by Aaron-Junker** [comment]
I guess the problem is that it installs itself in Excel and not as a seperate app.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-10-21 05:01:02.576 [CLI ] ShellExecute installer failed: 1603 
GetIsExcel14ClickToRunInstalled:  Error 0x80070002: Couldn't find registry key 'SOFTWARE\Microsoft\Office\14.0\CVH\Click2runExcel' Property(S): ErrorDialog = CustomErrorDlg 
MSI (s) (10:A8) [05:01:02:370]: Product: Microsoft Power Query for Excel (x64) -- Installation failed. 
MSI (s) (10:A8) [05:01:02:370]: Windows Installer installed the product. Product Name: Microsoft Power Query for Excel (x64). Product Version: 2.62.5222.701. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 1603. 

1603 usually means a missing dependency.

**Comment by stephengillie** [comment]
From the Application Log - Manual Validation ended with:
> 4:31:52 PM: Product: Microsoft Power Query for Excel (x64) -- Microsoft Power Query for Excel (x64) requires the 64-bit version of Microsoft Excel 2010 SP1 or Microsoft Excel 2013. Please first install the 64-bit version of Microsoft Excel 2010 SP1 or Microsoft Excel 2013.

File: manifests/m/Microsoft/PowerQuery/2.62.5222.761/Microsoft.PowerQuery.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.PowerQuery
+PackageVersion: 2.62.5222.761
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{2597FF8C-1E86-4793-8675-0CFC3A9094B6}'
```
Comment (stephengillie) [inline manifests/m/Microsoft/PowerQuery/2.62.5222.761/Microsoft.PowerQuery.installer.yaml:8]:
```suggestion
ProductCode: '{2597FF8C-1E86-4793-8675-0CFC3A9094B6}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.Office
```

**Comment by stephengillie** [comment]
With the dependency, Manual Validation ended with:
> 4:36:29 PM:  > 2023-10-23 16:36:29.418 [CLI ] Package hash verification failed. SHA256 in manifest [74b91e528ed9f80bf8a799b42045b8c9bb6758cde2c5a6efb2bda387b3ec0071] does not match download [caf295438040eca0632d475eeecf51a16307a23cb87dcfce796d274e8e8ff221]

File: manifests/m/Microsoft/PowerQuery/2.62.5222.761/Microsoft.PowerQuery.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: Microsoft.PowerQuery
+PackageVersion: 2.62.5222.761
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{2597FF8C-1E86-4793-8675-0CFC3A9094B6}'
+Installers:
+- InstallerUrl: https://download.microsoft.com/download/E/C/E/ECEA1718-B83D-4406-B49D-3488EA5800EC/PowerQuery_2.62.5222.761%20(64-bit)%20[en-us].msi
+  Architecture: x64
+  InstallerSha256: 0DC963482D42066813C1FC81475D425D7EC615D1FC30958480D8F1BDFB1244E4
```
Comment (stephengillie) [inline manifests/m/Microsoft/PowerQuery/2.62.5222.761/Microsoft.PowerQuery.installer.yaml:12]:
```suggestion
  InstallerSha256: CAF295438040ECA0632D475EEECF51A16307A23CB87DCFCE796D274E8E8FF221
```

**Comment by Aaron-Junker** [comment]
@stephengillie Now the hash is wrong

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   0DC963482D42066813C1FC81475D425D7EC615D1FC30958480D8F1BDFB1244E4
Expected hash: caf295438040eca0632d475eeecf51a16307a23cb87dcfce796d274e8e8ff221

**Comment by stephengillie** [comment]
The hash has rotated twice while trying to validate this PR. The team which manages this might release frequently, so might not be the best fit for the WinGet package manager at this time.

**Comment by Aaron-Junker** [comment]
> The hash has rotated twice while trying to validate this PR. The team which manages this might release frequently, so might not be the best fit for the WinGet package manager at this time.

This shouldn't be possible, as it's downloading the installer for this specific version

**Pull request closed without merging**